src/cadenza.js

```diff
@@ -868,24 +879,23 @@ export class CadenzaClient {
       resolvePath(source) + "/objectinfo" ,
       params,
       signal,
-      JSON.stringify({ objectIds, layerPath: array(layerPath), useMapSrs, fullGeometries}),
-      "application/json");
+      JSON.stringify({ objectIds, layerPath: array(layerPath), useMapSrs, fullGeometries}))
+      .then(response => response.json());
   }
 
   async #fetch(
     /** @type string */ path,
     /** @type URLSearchParams */ params,
     /** @type AbortSignal | undefined */ signal,
     /** @type String | undefined If body is set, the fetch will be a post.*/ body,
-    /** @type String | undefined Has to be set if body is set.*/ bodyContentType,
   ) {
     const url = this.#createUrl(path, params);
     this.#log('Fetch', url.toString());
     const method = body ? "POST" : undefined;
     const headers = new Headers();
     headers.set('X-Requested-With', 'XMLHttpRequest');
-    if (bodyContentType) {
-      headers.set('Content-Type', bodyContentType);
+    if (body) {
+      headers.set('Content-Type', "application/json");
```

Comment:
You still do not use the development watch, right? The CI will not allow to merge this.